grails-app/services/org/bbop/apollo/FeatureService.groovy

```diff
@@ -2594,68 +2660,127 @@ public void setTranslationEnd(Transcript transcript, int translationEnd) {
-                    it = Feature.get(it.id)
-                    it instanceof Gene
-                }
-
-                log.debug "overlapping features: ${overlappingFeatures.size()}"
-                for (Feature eachFeature : overlappingFeatures) {
-                    // get the proper object instead of its proxy, due to hibernate lazy loading
-                    Feature feature = Feature.get(eachFeature.id)
-                    log.debug "Evaluating overlap of feature ${feature.name} of class ${feature.class.name}"
-
-                    if (!gene && feature instanceof Gene && !(feature instanceof Pseudogene)) {
-                        Gene tmpGene = (Gene) feature
-                        log.debug "Found an overlapping gene: ${tmpGene}"
-                        Transcript tmpTranscript = (Transcript) convertJSONToFeature(jsonFeature, sequence)
-                        updateNewGsolFeatureAttributes(tmpTranscript, sequence)
-                        if (tmpTranscript.fmin < 0 || tmpTranscript.fmax < 0) {
-                            throw new AnnotationException("Feature cannot have negative coordinates")
-                        }
-
-                        setOwner(tmpTranscript, user)
-
-                        if (suppressHistory) {
-                            String name = nameService.generateUniqueName(tmpTranscript, tmpGene.name)
-                            tmpTranscript.name = name + "-" + tmpTranscript.alternateCvTerm
-                        }
-
-                        if (overlapperService.overlaps(tmpTranscript, tmpGene)) {
-                            log.debug "There is an overlap, adding to an existing gene"
-                            transcript = tmpTranscript;
-                            gene = tmpGene
-                            addTranscriptToGene(gene, transcript)
-                            transcript.save()
-                            // insert is false because this gene is already an existing gene
-                            gene.save(insert: false, flush: true)
-                            break
-                        } else {
-                            // deleting the temporarily generated transcript
-                            featureRelationshipService.deleteFeatureAndChildren(tmpTranscript)
-                            log.debug "There is no overlap, we are going to return a NULL gene and a NULL transcript"
-                        }
-                    } else {
-                        log.info "Feature is not an instance of a gene or is a pseudogene"
-                    }
-                }
+                // Scenario II - find and overlapping isoform and if present, add current transcript to its gene
+                // Disabling Scenario II since there is no appropriate overlapper to determine overlaps between non coding transcripts
+//                FeatureLocation featureLocation = convertJSONToFeatureLocation(jsonFeature.getJSONObject(FeatureStringEnum.LOCATION.value), sequence)
```

Comment:
I wonder if we can move this to another method and just comment out the single method.  If not, I think that just removing it with a note will be a bit better.